src/interfaces/utils-interfaces/form-field-type.ts

```diff
@@ -0,0 +1,6 @@
+import { FormType } from './form-type'
+
+export interface FormFieldType extends FormType {
```

Comment:
At one point I used this (ie, for $refs), but then didn't need it. But it can still be useful so I left it here.
Note that form field refs have properties that the form ref doesn't.